### PR TITLE
Infinite scrolling created

### LINKED ISSUE #777: Infinite scrolling
### What's the Play about?

An app, where user will scroll through the posts. on scrolling when user sees the last post, play will add more 20 posts to list.

### What ReactJS concept will be used to create this play?

Hooks , Props, Async-await, IntersectionObserver

### Are you willing to take it up for implementation?

Yes

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by aks318**
> First thing, PLEASE READ THIS: [ReactPlay Code Review Checklist](https://github.com/reactplay/react-play/wiki/ReactPlay-Code-Review-Checklist)

# Description
Infinite Scrolling Play Added #777 

Fixes #777 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

# Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by atapas** [comment]
@ammaaraslam Please review.

**Comment by aks318** [comment]
@ammaaraslam Please review it.

**Review by ammaaraslam — COMMENTED** [review]
@aks318 can you please add a cover image.

**Review by ammaaraslam — APPROVED** [review]
Everything looks good!

File: src/plays/infinite-scrolling/readme.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+- To observe the `lastPost` I have user `IntersectionObserver` and once it observe the `lastPost` it increments the `pageNo`.
+- And once the `pageNo` gets incremented, more 20 posts gets added into `displayPosts`
+
+## Consideration
+
+Update all considerations(if any)
+
+## Resources
+
+Update external resources(if any)
```
Comment (koustov) [inline src/plays/infinite-scrolling/readme.md:34]: Remove sections if not required

**Review by koustov — APPROVED** [review]
Conditional approval. Have one tiny comment.

**Comment by aks318** [comment]
@koustov Removed

**Comment by atapas** [comment]
Hi @aks318 

Please take care of the responsiveness

image

**Comment by aks318** [comment]
image
@atapas  kindly check

**Pull request merged**